src/app.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const { join } = require('path');
+const router = require('./routes/router');
+const {PORT} = require('dotenv');
+const app = express();
+
+app.use(express.json());
+app.use(express.urlencoded({extended:false}));
+app.use(express.static(join(__dirname,'..','public')));
+
+app.set('port',PORT || 5000);
+app.use(router);
+
+// app.get("/", (req, res)=>{
+//     console.log(ttt);
+// });
+
```

Comment:
It's a best practice to remove all console logs from production code to optimize performance and reduce clutter.